.github/workflows/release-tauri.yml

```diff
@@ -0,0 +1,89 @@
+name: Release Tauri
+
+on:
+    workflow_dispatch:
+        inputs:
+            environment:
+                type: environment
+                default: "Beta"
+            version:
+                type: string
+                description: "Version (semver) for the release."
+                default: "0.0.1"
+                required: true
```

Comment:
We can try to deduce this from the files, but it's not a bad idea to manually set it, since we'll be creating a tag from this. At least for the first release or so.

Reply:
It's not common practice, and I think a safer approach would be to confirm the versions in the code match what's being asked instead, but we can definitely have it update the files and commit, if that's easier.

Reply:
I think I'd like to try automatically updating the files in the repo, simply because that'll safe us more step and we can be absolutely sure the version is consistent everywhere. Can you think of any reason this may cause problems?

Reply:
Yes, it can more easily cause conflicts, and, for example, I can't test this without "adding noise" to the branch (though I can then force-push without that commit), but it shouldn't be a big problem, and I can appreciate the consistency ask.

I would, however, prefer to do it in a follow-up PR, once this script becomes a release for everything, if that's fine.